### ISSUE TITLE
Two strange Test failures on Windows

### ISSUE THREAD

**Issue by weshinsley** [mrc-ide/odin#173]
With R 3.5.3 on Windows 10, running on the gh-171 branch, I got two strange failures when testing within RStudio, and also externally with `make test` - but I could not emulate the failures by running the tests line by line.

```
test-odin-build.R:13: failure: build from validate
length(res$output) is not more than 1. Difference: -1

test-odin-build.R:31: failure: build from ir
length(res$output) is not more than 1. Difference: -1
```

When run manually, res$output at this point contains a few lines of echo from gcc; within testing it comes out as `character(0)`. odin_build.R:29-35 is the key part - does `utils::capture.output` behave differently when under "test" conditions, on Windows?

**Comment by weshinsley** [comment]
The failures are removed if I disable gcc warnings.

**Comment by richfitz** [comment]
Fixed in #174 - by skipping these tests on some platforms
